user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -59,6 +59,9 @@ Forge
 Others
 ------
 
+- **Database:** Added the ``numberNative`` attribute to the Database Config for MySQLi to keep the variable type obtained after SQL Query consistent with the type set in the database.
```

Comment:
Sorry, here is "Enhancements > Database > Others".

```suggestion
- **MySQLi:** Added the ``numberNative`` attribute to the Database Config to keep the variable type obtained after SQL Query consistent with the type set in the database.
```

Reply:
Thanks, I've made a correction in the next commit.